README.md

```diff
@@ -1,6 +1,33 @@
 # CodeSupport API
 
 ## About
-This repository contains the code for the CodeSupport API. The project is written in TypeScript using the [NestJS](https://nestjs.com/) framework.
+This repository contains the code for the CodeSupport API. The project is written in TypeScript using the [NestJS](https://nestjs.com/) framework with a postgreSQL database.
+
+## **Any Questions?**
+ Feel free to mention @LamboCreeper#6510 in the [CodeSupport Discord](https://codesupport.dev/discord).
```

Comment:
Can these lines be at the bottom of the README to keep consistent with our other repositories please.

Reply:
Done